src/_layouts/header.njk

```diff
@@ -16,5 +16,11 @@
     <div id="menu-reveal" class="menu-button-container right">
       Menu&nbsp;<i class="fa fa-bars"></i>
     </div>
+    <div class="menu">
+        <a title="Home" href="/" class="menu-item"></a>
+        <a title="About Us" href="/#about-us"></a>
+        <a title="Residential" href="/#residential"></a>
```

Comment:
this should ultimately look like `<a title="{$ link.description $}" href="{$ link.route $}" class="menu-item">{$ link.label $}</a>`